site/_posts/2017-03-07-deletions-and-dragons.html.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Last Week in ManageIQ: Deletions and Dragons"
+author: d-m-u
+date: 2017-03-07
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+## The Players 
+[Jamlamin Tarmikos][jt]: Male Human Dwarven Defender Str 12 Int 15 Wis 18 Dex 12 Con 16 Cha 17  
+[Thernda Strifelaugher][tw]: Female Halfling Dwarven Defender Str 15 Int 15 Wis 13 Dex 13 Con 6 Cha 11  
+[Janmorel][jc]: Male Half-Orc Loremaster Str 7 Int 15 Wis 12 Dex 7 Con 3 Cha 12  
+[Nerisstina Cupshigh][nc]: Female Gnome Paladin Str 14 Int 15 Wis 10 Dex 8 Con 13 Cha 9  
+[Krisnys Bearcharger][kb]: Female Elf Shadowdancer Str 4 Int 15 Wis 8 Dex 10 Con 5 Cha 19  
+[Eillyassa Chorster][ec]: Female Gnome Ranger Str 15 Int 15 Wis 3 Dex 3 Con 13 Cha 3  
+[Jannys Falconsflight][jf]: Female Half-Orc Paladin Str 6 Int 15 Wis 5 Dex 7 Con 11 Cha 4  
+[Janella Silentall][jr]: Female Half-Orc Dwarven Defender Str 11 Int 15 Wis 8 Dex 18 Con 11 Cha 13  
+[Magnys Gladomain][mh]: Female Elf Rogue Str 10 Int 15 Wis 13 Dex 10 Con 9 Cha 15  
+
+## Week Nine Overview
+
+Hello! Welcome to The Route of Elemental Code Deletions, a campaign where I, [Drew](https://github.com/d-m-u), will be DMing (Dungeon Mastering) a company of players as they attempt to navigate the treacherous module of Week Nine.
+
+Hungry and exhausted, our party hobbled along the mucky road that led outside Rahasia as they bickered fiercely over where to make camp. The past week had been particularly trying, with 139 total kills. This was [69 skeletons, trolls, ogres, and giants][manageiq PRs merged], [54 ancient bronze dragons, assorted demons, and devils][manageiq-ui-classic PRs merged], and [16 water weirds and elementals][manageiq-ui-service PRs merged]. 
+
+ [Thernda Strifelaugher][tw], a female Halfling Dwarven Defender, and [Jannys Falconsflight][jf], a female Half-Orc Paladin and aspiring acrobat, in particular, had had a rough go of it. They had battled respectively the ogre that was later found to be named [Vms API snapshots] [#13829] and a troll that subsequently revealed itself to be a [multiple parent issue on moving relationships][#14060]. Eventually the question of where to settle for the night became moot as what little light disappeared just as they had begun to cross a long bridge. They all stopped to pull out bedrolls, knowing that the best defense against the Dead was running water. [Jamlamin Tarmikos][jt], a male Human Dwarven Defender, woke with a start a few minutes later with a guilty twinge about the fact that no one was on guard. They had at one point had a system for who kept watch but the group dynamics had long since broken down and the trials of the week meant that any such orderly system had long been left by the wayside. He lay silent for a moment, until, with a start, he realized that he had heard something growl. Something with what sounded oddly like bells. He froze. 
+
```

Comment:
@d-m-u Not trying to break up the style of this write, but this has 4 had's in it and was hard for me to read:

`They had at one point had a system for who kept watch but the group dynamics had long since broken down and the trials of the week meant that any such orderly system had long been left by the wayside.`

I think you can write it in the same style with:

`They at one point had a system for who kept watch but the group dynamics had long since broken down and the trials of the week meant that any such orderly system had long been left by the wayside.`